src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx

```diff
@@ -14,7 +14,8 @@ import {PageContent, PageHeader} from 'app/styles/organization';
 import PageHeading from 'app/components/pageHeading';
 import BetaTag from 'app/components/betaTag';
 import space from 'app/styles/space';
-import {getStatus} from '../utils';
+
+import Status from '../status';
```

Comment:
Should this be an absolute import? What kind of guidelines should we have for when to use a relative vs absolute import?

Reply:
I've always been using relative when it's part of that view (e.g. this is part of the shared incident utils)